Tests/TestHelpers.swift

```diff
@@ -220,3 +220,14 @@ extension ImageType {
         }
     #endif
 }
+
+// A fixture for testing Decodable mapping
+struct Issue: Codable {
+    let title: String
+    let createdAt: Date
+
+    enum CodingKeys: String, CodingKey {
```

Comment:
@devxoul are these keys needed? I thought `Codable` automatically synthesized them if they're omitted.

Reply:
Yes, they should be automatically created and specifying the keys is only necessary if you want to change the name.

Reply:
Ah, didn't know that. Fixed!